src/Cardano/Shell/Constants/PartialTypes.hs

```diff
@@ -49,17 +49,12 @@ data PartialCardanoConfiguration = PartialCardanoConfiguration
 -- | Partial @Core@ configuration.
 data PartialCore = PartialCore
     { pcoGenesisFile                :: !(Last FilePath)
-    -- ^ Genesis source file JSON.
     , pcoGenesisHash                :: !(Last Text)
-    -- ^ Genesis previous block hash.
-    , pcoStaticKeySigningKeyFile    :: !(Last FilePath)
-    -- ^ Static key signing file.
-    , pcoStaticKeyDlgCertFile       :: !(Last FilePath)
-    -- ^ Static key delegation certificate.
+    , pcoStaticKeySigningKeyFile    :: !(Last (Maybe FilePath))
+    , pcoStaticKeyDlgCertFile       :: !(Last (Maybe FilePath))
     , pcoRequiresNetworkMagic       :: !(Last RequireNetworkMagic)
-    -- ^ Do we require the network byte indicator for mainnet, testnet or staging?
     , pcoDBSerializeVersion         :: !(Last Integer)
-    -- ^ Versioning for values in node's DB.
+    , pcoPBftSigThd                 :: !(Last (Maybe Double))
```

Comment:
This just looks weird.

Reply:
Yeah it doesn't need to be this way. It's already optional. It just means when we finalise the config, it's ok if it has no default value, we convert it into a `Maybe` at that point. Feel free to follow-up to fix this.

Reply:
Will do.

Reply:
The idea was to have a `Maybe` in the final `CardanoConfiguration`.  Now I'm struggling to remember why I thought it was needed.

Reply:
Undoing in https://github.com/input-output-hk/cardano-shell/pull/246.

Reply:
Yes, I remember now!

https://github.com/input-output-hk/cardano-shell/pull/246#issuecomment-518907922 is the explanation for `Maybe Double` in `coPbftSigThd`!